RELEASE-NOTES.txt

```diff
@@ -3,7 +3,7 @@
 15.0
 -----
 - [*] The store name can now be updated from the Settings screen. [https://github.com/woocommerce/woocommerce-ios/pull/10485]
-
+- [**] The store creation flow has been optimized to start store creation immediately and show profiler questions afterward. [https://github.com/woocommerce/woocommerce-ios/issues/10374]
```

Comment:
Should we replace the issue link with PR link/s instead?

Reply:
There are so many links, I don't think one would be enough. Please feel free to suggest the links you think is suitable for this.

Reply:
I agree. Maybe https://github.com/woocommerce/woocommerce-ios/pull/10473, https://github.com/woocommerce/woocommerce-ios/pull/10466 and https://github.com/woocommerce/woocommerce-ios/pull/10485.

Please also feel free to ignore my suggestion and keep the original issues link.

Reply:
I added an item for the store name item in Settings screen already so I'll just add the other two links then. Thanks!